logos/benchmarks/benchmark_logos.py

```
@@ -0,0 +1,1495 @@
+        # Base64-encode the poller script to avoid all shell quoting issues
+        script = _REMOTE_POLLER.format(gpu_index=self._gpu_index, interval=self._poll_s)
+        encoded = base64.b64encode(script.encode()).decode()
+        cmd = f"echo {encoded} | base64 -d | python3 -u"
+
+        ok_hosts = 0
+        for host in self.hosts:
+            samples: list[_RSample] = []
+            lock = threading.Lock()
+            try:
+                client = _paramiko.SSHClient()
+                client.set_missing_host_key_policy(_paramiko.AutoAddPolicy())
```

Ruff (0.15.15)

[error] Paramiko call with policy set to automatically trust the unknown host key

(S507)